website/content/docs/nia/usage/run-ha.mdx

```diff
@@ -0,0 +1,184 @@
+---
+layout: docs
+page_title: Run Consul-Terraform-Sync with High Availability
+description: >-
+  Improve network automation resiliency by enabling high availability for Consul-Terraform-Sync. HA enables persistent task and event data so that CTS functions as expected during a failover event.
+---
+
+# Run Consul-Terraform-Sync with High Availability
+
+This topic describes how to run Consul-Terraform-Sync (CTS) configured for high availability. High availability is an enterprise capability that ensures that all changes to Consul that occur during a failover transition are processed and that CTS continues to operate as expected.
+
+## Introduction
+
+A network always has exactly one instance of the CTS cluster that is the designated leader. The leader is responsible for monitoring and running tasks. If the leader fails, CTS triggers the following process if it is configured for high availability:
```

Comment:
```suggestion
If CTS is configured with high availability, a network always has exactly one instance of the CTS cluster that is the designated leader. The leader is responsible for monitoring and running tasks. If the leader fails, CTS triggers the following process:
```

To clarify, the leader / follower construct only exists when HA is configured. I wasn't sure if readers can mis-interpret the last sentence and believe that they can have a leader and high availability not configured